ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-community-lists.j2

```diff
@@ -7,15 +7,19 @@
 | Name | Action | Communities / Regexp |
 | ---- | ------ | -------------------- |
 {%     for community_list in ip_community_lists %}
-{%         for entry in community_list.entries %}
-{%             if entry.regexp is arista.avd.defined %}
+{%         if community_list.name is arista.avd.defined %}
+{%             for entry in community_list.entries %}
```

Comment:
Please add arista.avd.default([]) for entries. Similar to what you did in eos template.

Reply:
thx, forgot this one